src/script/modal.js

```
@@ -0,0 +1,24 @@
+const menu = document.getElementById('header-menu');
+const nav = document.getElementById('header-nav');
+const menuIcon = document.getElementById('icon-menu');
+
+menu.addEventListener('click', () => {
+    let isMenuOpen = menuIcon.classList.contains('icon-close');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'isMenuOpen' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)